assets/src/edit-story/panels/size/useSize.js

```diff
@@ -0,0 +1,59 @@
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+function useSize( width, height ) {
+	const [ lockRatio, setLockRatio ] = useState( true );
```

Comment:
I still think that this variable plus the `handleToggleLockRatio` could be moved to its own hook. Maybe even a more generalized hook for `useToggle` or `useBoolean`. It could then return an array, to allow easy renaming when using and destructing it (like `useState` does).

So it would be something like:

```js
const [ lockRatio, toggleLockRatio ] = useToggle( true );
```

It might be too trivial of a hook, maybe? Thoughts?

Reply:
I think it's good idea. Maybe we can define useToggle as common library like app folder.
And return array for any renaming in the future. That will avoid duplicated code for any toggle or flag values

Reply:
It makes sense. Create a helper in `/utils/useToggle` and put it there. It's very simple, but elegant. You could then even add a very simple unit test for it.

Reply:
okay